### ISSUE TITLE
nested-changes event not fire.

### ISSUE THREAD

**Issue by SurendraBorana-GSLab** [dhruvaray/backbone-associations#131]
Bind the event on model with nested-changes. If dynamically adding property after binding mention event, it is not fire.

**Comment by dhruvaray** [comment]
Nested-change event is switched off be default on the latest. You need to turn it on explicitly. Can you share a snippet of what you are tring to achieve?

**Comment by SurendraBorana-GSLab** [comment]
I enable the nested-change event using Backbone.Associations.EVENTS_NC = true;
I created  BackupModel with attribute backupId, backupname, sourceAddress, destinationAddress,...
Created object of BackupModel and bind the all property change event on same.

After backup initialize , some dynamic attribute added like percentage and backup status.
I want to monitor this two attribute change event, but this attribute change event not fire,,

I used the following code.
Backbone.Associations.EVENTS_NC = true;
var backupModel = new BackupModel();
backupModel.on('nested-change',function(){
 //some code here...
//THIs code never called
});
